src/framework/core/js/ModelTransformationTransforms.js

```diff
@@ -409,15 +433,19 @@ var fluid = fluid || fluid_2_0_0;
         }
 
         var outputArr = [];
-        fluid.each(options, function (arrVal, inPath) {
-            var val = fluid.model.transform.getValue(inPath, undefined, transformer);
+        fluid.each(input, function (val, key) {
             if (val === transformSpec.presentValue) {
-                outputArr.push(arrVal);
+                outputArr.push(options[key]);
```

Comment:
Was this simply a blunder in the original impl? This looks like a pretty significant change